src/lavatory/commands/policies.py

```diff
@@ -20,11 +20,13 @@
     multiple=True,
     required=False,
     help='Name of specific repository to run against. Can use --repo multiple times. If not provided, uses all repos.')
-def policies(ctx, policies_path, repo):
+@click.option('--repo-type', default='local', required=False, type=click.Choice(['local', 'virtual', 'cache', 'any']),
```

Comment:
I wonder if you should show the default when someone uses `--help`.
I believe the option is `show_defaults=True`.

Reply:
ohhh, I like that. I'll update the commands